pkg/minishift/clusterup/clusterup.go

```diff
@@ -275,7 +276,10 @@ func applyAddOns(addOnManager *manager.AddOnManager, ip string, routingSuffix st
 	return nil
 }
 
-func getConfigClusterName(ip string, port int) string {
+func getConfigClusterName(hostname string, ip string, port int) string {
+	if hostname != "" {
+		return "local-cluster"
+	}
 	return fmt.Sprintf("%s:%d", strings.Replace(ip, ".", "-", -1), port)
```

Comment:
but if set, then what?

Reply:
@gbraad if set then it will return "local-cluter" because that something part of cluster up code : https://github.com/openshift/origin/blob/master/pkg/oc/bootstrap/docker/openshift/login.go#L56

Reply:
@praveenkumar We should have added a comment that `cluster up` 's context it is local-cluster and we depend on `cluster up` for the kubeconfig.